src/country-by-abbreviation.json

```diff
@@ -251,10 +251,6 @@
         "country": "El Salvador",
         "abbreviation": "SV"
     },
-    {
-        "country": "England",
-        "abbreviation": null
-    },
```

Comment:
Thanks for the PR, but why is this removed? I know `null` is not the correct value, but you removed it from the file instead of fixing it I thin.

Reply:
England doesn't appear to be a valid value if you look a the Wikipedia page.

Reply:
Scotland was also removed but Northern Ireland was not, which is curious since they are peers in the same collection of countries (source: https://en.wikipedia.org/wiki/Countries_of_the_United_Kingdom)

According to the ISO_3166-1-alpha2 Wikipedia page, the correct country name is "United Kingdom of Great Britain and Northern Ireland".  You really cannot apply this data literally and end up with something useful.   Should all other variants be removed?

You end up in a philosophical quagmire of whether "Britain", "England", "Scotland", "Great Britain", "Northern Ireland", "Wales" are actual/useful country names or not.  My guess is that technically they are not according to the ISO, however they certainly are in common public opinion.  Is this a list of official data, or a useful list that helps people map commonly entered names?  The answer will decide the fate of "England".

Reply:
So, what should be done in this case? I am a bit clueless and constrained with time. I think I will merge and re-evaluate after and make improvements

Reply:
Well in this case it depends on what the data is being used for.

If the data is being used to map commonly known country names to their ISO 3166-1 Alpha-2 designations then you probably want to keep "England", "Scotland", "Wales", "Northern Ireland", "Britain" which are all commonly used aliases for "Great Britain" which is GB.

However the filename suggests "country by abbreviation" which would make me think the purpose of the data is to map the 2-letter country codes to country names.  If that's the case, there should only be one unique entry per "abbreviation", and the country name should be the official ISO 3166-1 designation perhaps.

I just found this today so I don't know what use cases the data is supposed to be for.